plugin.video.wakanim/resources/lib/list.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
```

Comment:
**Don't** ever use names that shadow Python built-in names like `list` and such! Please change the name of the module.